dist/css.js

```diff
@@ -6,6 +6,22 @@ const PREFERS_COLOR_SCHEME_REGEXP =
 
 const bodies = $WPT_BODIES;
 
+function countExternalCssInHead() {
+  return document.querySelectorAll( 'head link[rel="stylesheet"]' ).length;
+}
+
+function countInlineCssInHead() {
+  return document.querySelectorAll( 'head style' ).length;
+}
+
+function countExternalCssAfterHead() {
+  return document.querySelectorAll( 'link[rel="stylesheet"]' ).length - countExternalCssInHead();
```

Comment:
Why not just use the same as above but with `body`? In which case should also change the function name.

```suggestion
function countExternalCssInBody() {
  return document.querySelectorAll( 'body link[rel="stylesheet"]' ).length;
```

Reply:
Good point, fixed in https://github.com/HTTPArchive/custom-metrics/pull/64/commits/70a1cda990ffc87c50bca29342656b9690dda48e.